query/src/main/java/org/infinispan/query/impl/InfinispanQueryStatisticsInfo.java

```diff
@@ -18,7 +18,7 @@
  * @author anistor@redhat.com
  * @since 6.1
  */
-@MBean(objectName = "Statistics")
+@MBean(objectName = "Statistics", description = "Query statistics for the Hibernate Search integration")
```

Comment:
I wouldn't expose "Hibernate Search" in the name as it should be an implementation detail

Reply:
It is an implementation detail that we've always exposed. This MBean is defined by a HS interface. We had to create our implementation for it due to technical reasons. This will surely change drastically when moving to HS 6. 

Now the thing is @gustavonalle added some more stats here with ISPN-10884, which probably do not belong here.